### PR TITLE
Publish scp-ingest-pipeline to PyPI (SCP-2471)

### PR DESCRIPTION
**Pull request by jlchang**
Minor updates to publish scp-ingest-pipeline to PyPI

These changes support SCP-2471.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,8 +1,14 @@
 from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
 setup(
-    name='scp_ingest_pipeline',
-    version='0.1',
+    name='scp-ingest-pipe',
+    version='1.3.6rc1',
```
Comment (jlchang) [inline setup.py:8]: The repo is currently on release 1.3.6
The state of master has code that will be in release 1.3.7 - should our version be 1.3.6rc1 or 1.3.7rc1?

Comment (eweitz) [reply setup.py:8]: `1.3.7rc1`, I think.

Also, `scp-ingest-pipeline` instead of `scp-ingest-pipe`.

File: setup.py [hunk 2]
```diff
@@ -24,8 +30,15 @@
...
         'opencensus-ext-stackdriver',
         'google-cloud-trace',
-	'grpcio',
-	'grpcio-tools'
+        'grpcio',
+        'grpcio-tools',
     ],
     packages=find_packages(),
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```
Comment (devonbush) [inline setup.py:38]: nice, I think this is right -- not beta, but also not 'mature' :)

File: setup.py (hunk 1 above)
Comment (jlchang) [reply setup.py]: Thanks for catching scp-ingest-pipe! Fixed and using 1.3.7rc1 in `9a25885`

**Pull request merged**